ecosystem/sep-0012.md

```diff
@@ -48,6 +49,82 @@ Alternatively, if the client cannot add the authorization header. The JWT should
 ?jwt=<token>
 ```
 
+## Customer GET
+
+Checks the status of a customers information. This allows clients to check whether the customers information was accepted, rejected, or still needs more info.  Lookups are done via the stellar account ID and memo that were used for the initial PUT request.
+
+This also gives you the information needed as far as what information a KYC provider requires to KYC a user.  If you fetch an account that has never been KYC'd the server should respond with `NEEDS_INFO` and a list of all the fields you will need to provide.
+
+### Request
+
+```
+GET [KYC_SERVER]/customer?stellar_account_id=<public_key>&memo=<memo>
+```
+
+Name | Type | Description
+-----|------|------------
+`account` | `G...` string | The Stellar account ID used to identify this user
+`memo` | string | (optional) Unique identifier for user being identified, necessary for custodial, multi-user accounts.
+`memo_type` | string | (optional) type of `memo`. One of `text`, `id` or `hash`
+`lang` | string | (optional) Defaults to `en`.  Language code specified using [ISO 639-1](https://en.wikipedia.org/wiki/ISO_639-1).  Human readable descriptions and reasons should be in this language.
+
+### Response
+
+Name | Type | Description
+-----|------|------------
+`status` | string | Status of the customers KYC process.
+`fields` | array<object> | An array of fields still needed to finish KYC Process
+`reason` | string | Human readable reason for rejection
+
+```js
+// The case when a user has been successfully KYC'd and approved
+{
+   "status": "ACCEPTED",
+}
+```
+
+```js
+// The case when an anchor requires more info a bout a user
+{
+   "status": "NEEDS_INFO",
+   "fields": [
```

Comment:
So in SEP-31 `/info` the format for describing required fields is different. Most notably, the field name is a key. We don't _need_ to have the same format but I think its better if it does.

I like the `type` attribute here though, maybe we can add that to the SEP-31 `/info` endpoint. But I would definitely add the `optional` and `choices` fields to this schema.

Reply:
Good call, updated.